code/style.css

```diff
@@ -1,5 +1,465 @@
-/* After you've linked the CSS file in the HTML,
-this should turn the background blue ;) */
+@import url('https://fonts.googleapis.com/css2?family=Cinzel:wght@400..900&display=swap');
```

Comment:
Since you already added this font in the HTML tag via `<link>` in the `<head>` section you do not need to add it here as well. Generally I think `<link>` is the recommended way to go.